models/Program.ts

```diff
@@ -1,7 +1,8 @@
 import type { programFormat } from "@prisma/client";
 
 /**
- * Request Body Input for POST /program
+ * custom type used for the POST endpoint when creating a new program
+ * id is not required and is automatically incremented in the record
```

Comment:
We can actually get rid of isArchived when creating a program because it will always be false. I think it is default false in the database already.

Reply:
Good call